webapp/src/modules/rental/sagas.spec.ts

```diff
@@ -505,11 +509,19 @@ describe('when handling the request action to claim a LAND', () => {
               ),
               Promise.resolve(txHash)
             ],
-            [call(waitForTx, txHash), Promise.resolve()]
+            [call(waitForTx, txHash), Promise.resolve()],
+            [
+              call(waitUntilRentalChangesStatus, nft, RentalStatus.CLAIMED),
+              Promise.resolve()
+            ],
+            [
+              call(waitUntilRentalChangesStatus, nft, RentalStatus.CLAIMED),
+              Promise.resolve()
+            ],
```

Comment:
I think that one of them is repeated:
```suggestion
            ],
```

Reply:
ohh right 🤦 thanks! I just removed one